src/lib/accessibility/Behaviors/List/selectableListItemBehavior.ts

```diff
@@ -11,6 +11,7 @@ const selectableListItemBehavior: Accessibility = (props: any) => ({
     root: {
       role: 'option',
       'aria-selected': !!props['active'],
+      tabIndex: props.focusableItemProps.isFocused ? '0' : '-1',
```

Comment:
seems that we should at least specify type for `props` here, so that it will be clear from which context `focusableItemProps` come from

Reply:
For the next round of refactoring